<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>506</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文楷体</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>画图</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/tb.png</normaloff>:/new/prefix1/images/tb.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>文件（F）</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_O"/>
    <addaction name="separator"/>
    <addaction name="action_S"/>
    <addaction name="action_A"/>
    <addaction name="separator"/>
    <addaction name="action_X"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>编辑（E）</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>工具（T）</string>
    </property>
    <addaction name="action_14"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>帮助（H）</string>
    </property>
    <addaction name="action_15"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_O"/>
   <addaction name="action_S"/>
   <addaction name="separator"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
   <addaction name="action_7"/>
   <addaction name="separator"/>
   <addaction name="action_13"/>
   <addaction name="action_14"/>
   <addaction name="action_15"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>83</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>绘图工具</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>直线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>矩形</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>椭圆</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>实线</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>点线</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="penColorComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="brushColorComboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选择图形</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>画笔类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>画笔宽度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>画笔颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>填充颜色</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>64</number>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/new.png</normaloff>:/new/prefix1/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="statusTip">
    <string>新建文件夹</string>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/open.png</normaloff>:/new/prefix1/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="statusTip">
    <string>打开图像文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/save.png</normaloff>:/new/prefix1/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string>保存当前绘制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_A">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/saveas.png</normaloff>:/new/prefix1/images/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存</string>
   </property>
   <property name="statusTip">
    <string>将当前绘制另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_X">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/exit.png</normaloff>:/new/prefix1/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/zoom-in.png</normaloff>:/new/prefix1/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="statusTip">
    <string>放大画面</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/zoom-out.png</normaloff>:/new/prefix1/images/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="statusTip">
    <string>缩小画面</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/zoom-1.png</normaloff>:/new/prefix1/images/zoom-1.png</iconset>
   </property>
   <property name="text">
    <string>还原</string>
   </property>
   <property name="statusTip">
    <string>原比例显示</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/shear.png</normaloff>:/new/prefix1/images/shear.png</iconset>
   </property>
   <property name="text">
    <string>扭曲</string>
   </property>
   <property name="statusTip">
    <string>拉伸图像</string>
   </property>
   <property name="shortcut">
    <string>'</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/clear.png</normaloff>:/new/prefix1/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除</string>
   </property>
   <property name="statusTip">
    <string>删除全部内容</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/tool.png</normaloff>:/new/prefix1/images/tool.png</iconset>
   </property>
   <property name="text">
    <string>绘图工具栏</string>
   </property>
   <property name="statusTip">
    <string>工具栏</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/guanyu.png</normaloff>:/new/prefix1/images/guanyu.png</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
   <property name="statusTip">
    <string>关于</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/rotate1.png</normaloff>:/new/prefix1/images/rotate1.png</iconset>
   </property>
   <property name="text">
    <string>顺时针旋转</string>
   </property>
   <property name="shortcut">
    <string>[</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/rotate2.png</normaloff>:/new/prefix1/images/rotate2.png</iconset>
   </property>
   <property name="text">
    <string>逆时针旋转</string>
   </property>
   <property name="shortcut">
    <string>]</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/shear.png</normaloff>:/new/prefix1/images/shear.png</iconset>
   </property>
   <property name="text">
    <string>扭曲</string>
   </property>
   <property name="shortcut">
    <string>\</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
